oca_dependencies.txt

```diff
@@ -14,24 +14,24 @@
 # To provide both the URL and a branch, use:
 
 # Uncomment when needed
-account-financial-tools
-bank-payment
+account-financial-tools 11.0
+bank-payment 11.0
```

Comment:
Don't put the branch name for OCA repos! It is only needed for our forks, if the branch name is not `11.0`